<launch>
    <arg name="robot_name" />
    <arg name="goal_x" default="0.0"/>
    <arg name="goal_y" default="0.0"/>
    <arg name="orientation_w" default="1.0"/>

    <node name="$(arg robot_name)_move_base_mover_node" pkg="uml_hri_nerve_navigation" type="move_base_mover_node" output="screen">  
        <param name="robot_name" value="$(arg robot_name)"/>
        <param name="goal_x" value="$(arg goal_x)"/>
        <param name="goal_y" value="$(arg goal_y)"/>
        <param name="orientation_w" value="$(arg orientation_w)"/>
    </node>
</launch>
